src/services/player.rs

```diff
@@ -92,11 +105,13 @@ fn handle_message<M: Messenger>(
         //When we get a message from a peer that comes from one of our anchored players we want to
         //make sure they don't get the result packets.
         PlayerStateOperations::BroadcastAnchoredEvent(msg) => {
-            trace!("Broadcasting Anchored Event for entity {:?}", msg.entity_id);
+            if let Some(entity_id) = entity_conn_ids.get(&msg.entity_id) {
+                trace!("Appending entity id {:?} to anchored event", entity_id);
+            }
             messenger.broadcast_packet(
                 msg.packet,
                 entity_conn_ids.get(&msg.entity_id).copied(),
-                true,
+                false,
```

Comment:
This was a bug that was revealed when mutual connections started working. Certainly events from anchored players are not local events that should be broadcast to every peer